src/flags.rs

```diff
@@ -0,0 +1,65 @@
+#[derive(Default)]
+pub struct X86Flags {
+	pub carry: bool,
+	pub parity: bool,
+	pub adjust: bool,
+	pub zero: bool,
+	pub sign: bool,
+	pub trap: bool,
+	pub interrupt: bool,
+	pub direction: bool,
+	pub overflow: bool,
+	pub iopl: bool,
+	pub nested_task: bool,
+	pub resume: bool,
+	pub virtual_8086_mode: bool,
+	pub alignment_check: bool,
+	pub id_capability: bool,
+	pub vif: bool,
+	pub vip: bool
+}
```

Comment:
vip, vif, id_capability, virtual_8086_mode, resume, nested_task, iopl, interrupt, and trap can be removed, as they'll never be used